t/lib/OpenQA/Test/Utils.pm

```diff
@@ -467,13 +465,14 @@ sub unstable_worker {
             });
         setup_worker($worker, $host);
         $worker->init();
+        Devel::Cover::report() if Devel::Cover->can('report');
         if ($ticks < 0) {
+            # This call will effectively never return
             Mojo::IOLoop->singleton->start;
         }
         else {
             Mojo::IOLoop->singleton->one_tick for (0 .. $ticks);
         }
-        Devel::Cover::report() if Devel::Cover->can('report');
         if ($sleep) {    # uncoverable statement
             1 while sleep $sleep;    # uncoverable statement
         }    # uncoverable statement
```

Comment:
likely this can be even simplified to

```suggestion
        1 while sleep $sleep;    # uncoverable statement
```

or

```suggestion
        sleep $sleep if $sleep;    # uncoverable statement
```

it seems one can even just do

```suggestion
        sleep $sleep;    # uncoverable statement
```